src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -158,6 +158,11 @@ private function doAsk(OutputInterface $output, Question $question)
         $inputStream = $this->inputStream ?: STDIN;
         $autocomplete = $question->getAutocompleterValues();
 
+        if (\function_exists('sapi_windows_cp_set')) {
+            // Codepage used by cmd.exe on Windows to allow special characters (éàüñ).
+            sapi_windows_cp_get(437);
```

Comment:
That doesn't look right. `function_exists` checks for `sapi_windows_cp_set`, but `sapi_windows_cp_get` is being used.